### PR TITLE
Display games on home page

### PR DESCRIPTION
**Pull request by halogenandtoast**

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class GamesController < ApplicationController
+  def initialize
+    @games = Game.limit(100)
```
Comment (CharlesMassry) [inline app/controllers/games_controller.rb:3]: Looks like a good candidate for pagination, although maybe another pull request.

File: app/views/games/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<ol>
+  <% @games.each do |game| %>
```
Comment (CharlesMassry) [inline app/views/games/index.html.erb:2]: Not a partial collection?

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (halogenandtoast) [inline app/controllers/games_controller.rb:3]: Not paginated, only top 100, at the very least once I add rank I'll order by that.

File: app/views/games/index.html.erb (hunk 2 above)
Comment (halogenandtoast) [inline app/views/games/index.html.erb:2]: Not if I want to wrap it in an li and also use it somewhere else.

**Pull request closed without merging**